lib/main/error/capability_validator.dart

```diff
@@ -37,4 +37,18 @@ extension CapabilityIdentifierExtension on List<CapabilityIdentifier> {
       return false;
     }
   }
+}
+
+extension CapabilityIdentifierSetExtension on Set<CapabilityIdentifier> {
+
+  Set<CapabilityIdentifier> toCapabilitiesSupportTeamMailboxes(Session session, AccountId accountId) {
+    try {
+      requireCapability(session, accountId, toList());
```

Comment:
Call `requireCapability` for capability support `Team-Mailbox`
```suggestion
requireCapability(session, accountId, [CapabilityIdentifier.jmapTeamMailboxes]);
```